src/SourceBuild/tarball/patches/roslyn/0005-Exclude-src-Dependencies-from-source-build.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Davis Goodin <dagood@microsoft.com>
+Date: Fri, 15 Oct 2021 02:41:32 -0500
+Subject: [PATCH] Exclude src/Dependencies from source-build
+
+This makes source-build stop producing the Microsoft.CodeAnalysis.Collections package. Downstream repos (MSBuild) can't safely use a source-built version.
+
+PR for applying this patch: https://github.com/dotnet/roslyn/pull/57165
+---
```

Comment:
Nit: I think this can be squashed into the first patch to simplify the overall diff. No need to have one patch that adds `Microsoft.CodeAnalysis.Collections` and a second one that removes it. (Especially now that we know adding `Microsoft.CodeAnalysis.Collections` to source-build is a non-starter.)

Reply:
Squashed these together in https://github.com/dotnet/installer/pull/12397/commits/83689bbca813a7fbcda47877337a3c7a8dd9dfb0